docs/architecture/cloud-native/scale-dotnet-applications.md

```diff
@@ -9,30 +9,29 @@ One of the most-often touted advantages of moving to a cloud hosting environment
 
```

Comment:
this article file name should probably be rename to just scale-applications.md to avoid repeating words in the path

Reply:
also, do you mind updating the description so it describes what this specific file is covering?